controller/api/management/commands/load_db_state_to_etcd.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from api.models import Key, App, Domain, Certificate, Config
+
+
+class Command(BaseCommand):
```

Comment:
There should be a docstring here describing what the `Command` class does, and another docstring for `handle`.